htlcswitch/interceptable_switch.go

```diff
@@ -406,6 +418,49 @@ func (s *InterceptableSwitch) resolve(res *FwdResolution) error {
 	}
 }
 
+// resignChannelUpdate signs the provided channel update with our node key.
+func (s *InterceptableSwitch) resignChannelUpdate(
+	update *lnwire.ChannelUpdate) error {
+
+	sig, err := s.signChannelUpdate(update)
+	if err != nil {
+		return err
+	}
+
+	update.Signature, err = lnwire.NewSigFromSignature(sig)
+	if err != nil {
+		return nil
+	}
+
```

Comment:
Doesn't commit it. Even if we do decide to allow this, we also need to make sure that the update would actually validate in the first place (has all the needed fields, bounds respected, etc).

Reply:
You mean commit as in write to disk? As explained in https://github.com/lightningnetwork/lnd/pull/7067/commits/e4af8d7a4856d0d6597a264aed74f0df8834a749#r1202002584, it is the intention to just pass through without persisting.